contrib/win32/openssh/sshd.vcxproj

```diff
@@ -136,7 +136,7 @@
     <Link>
       <SubSystem>Console</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
-      <AdditionalDependencies>posix_compat.lib;libssh.lib;openbsd_compat.lib;libcrypto-41.lib;$(AdditionalDependentLibs);%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>posix_compat.lib;libssh.lib;openbsd_compat.lib;libcrypto.lib;$(AdditionalDependentLibs);%(AdditionalDependencies)</AdditionalDependencies>
       <AdditionalLibraryDirectories>$(OpenSSH-Lib-Path)$(Platform)\$(Configuration);$(LibreSSL-x64-Path);%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
```

Comment:
How are the unit test binaries building for OneCore without the  (AdditionalDependentLibs);% change that we did for core binaries?

Reply:
IT looks like OneCore SDK includes Ws2_32.lib;Netapi32.lib;Secur32.lib - libs that unittests are linking against. 
 it will be good if we can change those to the model we have for core binaries for consistency.